packages/wix-ui-core/src/clients/Atlas/AtlasBasicClient.ts

```diff
@@ -0,0 +1,113 @@
+import {
+  CommonAddress,
+  V2GetPlaceResponse,
+  V2Prediction,
+  WixAtlasServiceWeb,
+} from '@wix/ambassador-wix-atlas-service-web/http'
+import {
+  Address,
+} from './types';
+import { BaseMapsClient } from '../GoogleMaps/types'
+
+const ATLAS_WEB_BASE_URL = '/api/wix-atlas-service-web';
+const BASE_LINGUIST_HEADER =
+  '|en-us|false|4e8a573a-6b3e-426f-9d2f-5285b7dc90f8';
+const { AutocompleteServiceV2, PlacesServiceV2 } = WixAtlasServiceWeb(ATLAS_WEB_BASE_URL);
+
+const serializeResult = (results: CommonAddress[]) =>
```

Comment:
```suggestion
const serializeGeocodeResult = (results: CommonAddress[]) =>
```
Maybe better to differentiate so it's easier to understand we're doing something different here.